packages/skeleton-common/src/classes/menu.ts

```diff
@@ -0,0 +1,17 @@
+import { defineSkeletonClasses } from '../internal/define-skeleton-classes.js' with { type: 'macro' };
+
+export const classesMenu = defineSkeletonClasses({
+	trigger: '',
+	contextTrigger: '',
+	positioner: '',
+	indicator: '',
+	content: 'card bg-surface-50-950 border border-surface-200-800 p-2 flex flex-col gap-2',
```

Comment:
Finally, I might consider adding a min-width. This feels way too thin. Try `min-w-3xs` as it's the smallest built-in width:

<img width="193" height="271" alt="Screenshot 2025-11-14 at 10 01 23 AM" src="https://github.com/user-attachments/assets/6e2ddc7f-d038-46bc-b0e0-e2600eafba15" />